src/components/LineChart/LineChart.js

```diff
@@ -114,100 +115,105 @@ function LineChart({
       delay={animDelay}
       reset={reset}
     >
-      {({ progress }) => (
-        <svg
-          ref={onSvgRef}
-          viewBox={`0 0 ${width} ${height}`}
-          width={widthProps || 'auto'}
-          height="auto"
-          css="display: block"
-          {...props}
-        >
-          <mask id="chart-mask">{rectangle}</mask>
-          {rectangle}
-
-          <g mask="url(#chart-mask)">
-            {totalCount > 0 && (
-              <path
-                d={`
+      {/* {({ progress }) => { */}
+      {({ progress }) => {
```

Comment:
`progress` here dont look be unsed, instead its using `val=1`, maybe was justing testing?

Reply:
Reverting the changes in the LineChart component